aim/web/ui/.eslintrc

```diff
@@ -29,6 +29,7 @@
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
     "react/react-in-jsx-scope": "off",
+    "react/display-name": "off",
```

Comment:
We are using display-name or our components in time of debugging.

Reply:
yes but when u want to create an arrow function which returns jsx it is impossible with this eslint config